eclipse-theia/theia
# Move TerminalShellIntegration stubs to main theia API

#### What it does
Moves already stubbed API from proposed to main API for TerminalShellIntegration
As developed in the issue #14107, implementation of the feature is a large task that needs to be planned and evaluated. Moving already stubbed API is a safe move for this month. 

Fixes #14107 

Contributed on behalf of STMicroelectronics

#### How to test
Install latest Python as described in PR #14058 (2024.12.3). Ensure that the extension works doesn't crash on startup. 
 
#### Follow-ups

Implement stubbed API. see task #14169 

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
